pkg/storage/segments/repairer_test.go

```diff
@@ -68,23 +70,36 @@ func TestSegmentStoreRepair(t *testing.T) {
 		remotePieces := pointer.GetRemote().GetRemotePieces()
 		minReq := redundancy.GetMinReq()
 		numPieces := len(remotePieces)
-		toKill := numPieces - (int(minReq) + 1)
+		// disqualify one storage node
+		toDisqualify := 1
+		toKill := numPieces - (int(minReq+1) + toDisqualify)
```

Comment:
Like in TestDataRepair, add a check that we kill at least one node. I am not sure this will pass with the 2/3/4/4 RS scheme in this test.

Reply:
it did pass!

Reply:
oh I was looking at the wrong thing.. no it didn't.

Reply:
well I updated the RS setting to be the same as TestDataRepair, but now I'm kind of wondering what the benefit is of having two separate tests that seem to do about the same thing. do you think it would be okay to remove one of them?

Reply:
These two tests really look to be the same. Let's remove the TestSegmentStoreRepair test.